A38-admin-interface-api.md

```diff
@@ -0,0 +1,200 @@
+Admin Interface API
+----
+* Author(s): lidiz
+* Approver: markroth
+* Status: Draft
+* Implemented in: C++, Golang, Java
+* Last updated: 2021-01-27
+* Discussion at: TBD
+
+## Abstract
+
+This proposal describes a convenience API in each gRPC language to improve the usability of creating a gRPC server with admin services.
+
+
+## Background
+
+gRPC Admin Interface is part of the Proxyless Service Mesh Observability project which includes new gRPC admin services and a CLI tool (yet to release). The admin services are responsible for exposing states of the gRPC library via gRPC protocol, and the CLI tool will help users to better interpret the fetched data from the gRPC application. Currently, the existing admin server should at least expose two admin services:
+
+* Channelz: the channel tracing library that collects stats about a channel’s traffic, underlying sockets, and credentials ([proto definition](https://github.com/grpc/grpc/blob/master/src/proto/grpc/channelz/channelz.proto));
```

Comment:
This can have a link to gRFC A14 (https://github.com/grpc/proposal/blob/master/A14-channelz.md).

Reply:
Done.